shell/src/main/java/org/apache/kafka/shell/glob/GlobVisitor.java

```diff
@@ -94,9 +94,9 @@ public String toString() {
     @Override
     public void accept(MetadataShellState state) {
         String fullGlob = glob.startsWith("/") ? glob :
-            state.workingDirectory() + "/" + glob;
+                state.workingDirectory() + "/" + glob;
```

Comment:
The reason to add this indent, maybe we should remain the same to keep the uniform format.